Dockerfile.NL

```diff
@@ -1,4 +1,4 @@
-FROM pdok/mapserver:7.6.4-patch5-buster-lighttpd
+FROM pdok/mapserver:7.6.4-patch5-3-buster-lighttpd
```

Comment:
```suggestion
FROM pdok/mapserver:7.6.4-patch3-buster-lighttpd
```

Reply:
there is already a image with that same tag

Reply:
```
$ docker pull pdok/mapserver:7.6.4-patch5-3-buster-lighttpd
Error response from daemon: manifest for pdok/mapserver:7.6.4-patch5-3-buster-lighttpd not found: manifest unknown: manifest unknown
```

Reply:
o, `pdok/mapserver:7.6.4-patch5-3-buster-lighttpd` will be created from `Dockerfile`